test/utils/update_resources.go

```diff
@@ -48,9 +48,16 @@ func ScaleResourceWithRetries(scalesGetter scaleclient.ScalesGetter, namespace,
 		Size:            -1,
 		ResourceVersion: "",
 	}
+	// timeouts per iterations: 5s, 10s, 20s, 40s...
+	exponentialBackoff := wait.Backoff{
```

Comment:
nit: should it be decorated with `const`?

Reply:
Not sure what do you mean?